docs/UserGuide.md

```diff
@@ -345,22 +321,120 @@ Examples:
+
+2. Employees who are can work on `Monday` or `Friday` will be listed in the _Employee Directory_ pane.
+
+{Screenshot}
+
+#### 3.6 Locating all employees who are available to work today: `c-today`
+
+Finds employees whose tags contain today's day (i.e. `Wednesday`, `Thursday`, etc).
+
+Format: `c-today`
+
+* The search is case-insensitive (e.g `Friday` will match `friday`).<br>
+* The application will automatically get today's day based on the computer's current date.<br>
+* Only full words will be marched (e.g `Fri` will not match `Friday`).<br>
+
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Note:**<br>
+* In tCheck, if an employee contains a tag for a specific day, it means this employee is available on that day
+. You may set an employee's tags when adding the employee into tCheck (Refer to `c-add` command), or edit the
+tag(s) by editing employee's information feature (Refer to `c-edit` command).<br>
+
+</div>
+
+Example:
+* Let's say today is `Thursday`. You want to find out who are the available manpower for today. You can follow these
+ steps.
+
+Steps to find today's available manpower:
+1. Type `c-today` into the _Command Box_.
+2. Press "Enter" to execute.
+
+Outcome:
+1. The _Result Display_ will show 3 information:
+    
+    a. Today's day. (e.g. Today is THURSDAY.)
+    
+    b. Total number of employees who are available today.
+    
+    c. The success message.
+2. The available manpower for today will be listed in the _Employee Directory_ pane.
+
+{Screenshot}
+
+#### 3.7 Locating all employees who are available to work tomorrow: `c-tomorrow`
+
+Finds employees whose tags contain tomorrow's day (i.e. `Wednesday`, `Thursday`, etc).
+
+Format: `c-tomorrow`
+
+* The search is case-insensitive. e.g `Friday` will match `friday`.<br>
+* The application will automatically get tomorrow's day based on the computer's current date.<br>
+* Only full words will be marched (e.g `Fri` will not match `Friday`).<br>
+
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Note:**<br>
+* In tCheck, if an employee contains a tag for a specific day, it means this employee is available on that day
+. You may set an employee's tags when adding the employee into tCheck (Refer to `c-add` command), or edit the
+tag(s) by editing employee's information feature (Refer to `c-edit` command).<br>
+
+</div>
+
+Example:
+* Let's say tomorrow is `Friday`. You want to find out who are the available manpower for tomorrow. You can follow these
+ steps.
+
+Steps to find tomorrow's available manpower:
+1. Type `c-tomorrow` into the _Command Box_.
+2. Press "Enter" to execute.
+
+Outcome:
+1. The _Result Display_ will show 3 information:
+    
+    a. Tomorrow's day. (e.g. Today is FRIDAY.)
+    
+    b. Total number of employees who are available tomorrow.
+    
+    c. The success message.
+2. The available manpower for tomorrow will be listed in the _Employee Directory_ pane.
```

Comment:
Same issue here.

Reply:
It has been changed in the latest commit. Thanks!